docs: add faq to woocommerce manual (#101)
myparcelnl/developer: src/nl/documentatie/10.woocommerce.md

## Veelgestelde vragen

> ### Ik wil als standaard vervoerder DHL inschakelen in plaats van PostNL.

Reviewer: ```suggestion
### Ik wil als standaard vervoerder DHL inschakelen in plaats van PostNL
```

Het is niet direct mogelijk om DHL als standaard vervoerder te kiezen. Door de bezorgopties van PostNL uit te schakelen, wordt alleen DHL aangeboden tijdens het afrekenen. Hierdoor wordt DHL de enige vervoerder die wordt doorgestuurd naar de MyParcel backoffice.
<MPImg src="/documentation/woocommerce/myparcel-postnl-bezorgoptie-inactief.jpg" alt="MyParcel PostNL bezorgopties inactief zetten" />

> ### Ik wil brievenbuspakje als standaard pakkettype naar de MyParcel backoffice doorschieten.

Reviewer: ```suggestion
### Ik wil brievenbuspakje als standaard pakkettype naar de MyParcel backoffice doorschieten
```

> ### Ik wil brievenbuspakket als standaard pakketType naar de MyParcel backoffice doorschieten.

Reviewer 2: ```suggestion
### Ik wil brievenbuspakje als standaard pakkettype naar de MyParcel backoffice doorschieten.
```

Je hebt een keuze uit 2 opties

<strong>Optie 1</strong>:<br>

> Indien een webshop alleen brievenbuspakketten verzendt, is het mogelijk om in WooCommerce één vast verzendmethode voor verzending aan te maken.

Reviewer 2: ```suggestion
Indien een webshop alleen brievenbuspakjes verzendt, is het mogelijk om in WooCommerce een vaste verzendmethode voor verzending aan te maken.
```

<MPImg src="/documentation/woocommerce/myparcel-default-pakkettype-bbp.jpg" alt="WooCommerce verzendmethode voor brievenbus pakket" />

> en in de MyParcel Plugin > Standaard exportinstellingen deze verzendmethode te koppelen aan pakkettype brievenbuspakket.

Reviewer 2: ```suggestion
en in de MyParcel Plugin > Standaard exportinstellingen deze verzendmethode te koppelen aan pakkettype brievenbuspakje.
```

<MPImg src="/documentation/woocommerce/myparcel-koppel-verzendmethode-brievenbuspakket.jpg" alt="WooCommerce verzendmethode koppelen aan brievenbus pakkettype" />

<strong>Optie 2</strong>:<br>

> Indien naast brievenbuspakketten ook andere pakkettypen worden verzonden, is het mogelijk op productniveau aan te geven of het product als brievenbuspakket verstuurd moet worden.

Reviewer 2: ```suggestion
Indien naast brievenbuspakjes ook andere pakkettypen worden verzonden, is het mogelijk op productniveau aan te geven of het product als brievenbuspakje verstuurd moet worden.
```
